index.html

```diff
@@ -1309,13 +1309,24 @@ <h2 id="privacy-considerations">
         [=permission=] [=permission/state=] of [=powerful features=] associated with an [=origin=].
       </p>
     </section>
+    <section id="accessibility-considerations">
+      <h2>
+        Accessibility considerations
+      </h2>
+      <p>
+        Interfaces intended to allow the user to  review, update, and reset the
+        [=permission=] [=permission/state=] of [=powerful features=] associated with an [=origin=].
+        MUST respect platform accessibility conventions.
```

Comment:
Isn't this true of every UA interface? Why would permissions need special text for this? Were there specific problems with permission UI? Did browsers push back when you filed bugs about those?

Reply:
ping @chaals

Reply:
I tend to agree with @jyasskin... this is seems somewhat overly general and doesn't really much. 

I'd be inclined to add guidance around this if we had seen issues (or foresee accessibility issues!) with the UIs being presented. 

So far, the UIs have been ok, no?